airflow/operators/mysql_operator.py

```diff
@@ -32,6 +32,8 @@ class MySqlOperator(BaseOperator):
     :type sql: Can receive a str representing a sql statement,
         a list of str (sql statements), or reference to a template file.
         Template reference are recognized by str ending in '.sql'
+    :param parameters: (optional) the parameters to render the SQL query with.
```

Comment:
Could you fix the order? :) The `sql` one comes before `mssql_conn_id`, but in the docstring it is the other way around.  Also `autocommit is missing here`.